core/src/test/resources/configs/xsite/xsite-test.xml

```diff
@@ -29,6 +29,26 @@
                     failure-policy-class="org.infinispan.xsite.CountingCustomFailurePolicy" timeout="160000"/>
          </backups>
       </distributed-cache>
+      <distributed-cache name="conflictResolver" configuration="base">
+         <backups conflict-resolver="PREFER_NON_NULL">
```

Comment:
For consistency should we also utilise `merge-policy` for xsite? The discrepency in the XML seems odd to me as both `conflict-resolve` and `merge-policy` have the same purpose, albeit in the different contexts:

```xml
<distributed-cache name="conflictResolver" configuration="base">
  <partition-handling when-split="ALLOW_READ_WRITES" merge-policy="PREFERRED_NON_NULL"/>
  <backups conflict-resolver="PREFER_NON_NULL">
    <backup site="NYC2" strategy="ASYNC"/>
  </backups>
</distributed-cache>
```

Reply:
done

Reply:
Sorry for mentioning this so late, but similarly we should probably make the conflict resolution interface SPIs consistent as well. Maybe `XSiteConflictResolver` -> `XSiteEntryMergePolicy`, similarly the `resolve` method can be changed to `merge`.

Reply:
done :)